distributions/Dist_yl4874.py

```diff
@@ -13,7 +14,7 @@ def __init__(self):
 
 	def pdf(self, x):
 
-		return np.abs(lnx) #normalizing lnx  in bounds 0,1 gives us the function abs(lnx)
+		return np.abs(math.log(x)) #normalizing lnx  in bounds 0,1 gives us the function abs(lnx)
```

Comment:
Sorry about this, but python's math.log function won't work with Numpy broadcasting. Use np.log.